### PR TITLE
Adding output binding for commercetools

### PR DESCRIPTION
**Pull request by harrykimpel**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [X] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Adding output binding for commercetools.

## Issue reference

n/a

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/commercetools.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+
+## Related links
+
+- [Basic schema for a Dapr component]({{< ref component-schema >}})
+- [Bindings building block]({{< ref bindings >}})
+- [How-To: Trigger application with input binding]({{< ref howto-triggers.md >}})
+- [How-To: Use bindings to interface with external resources]({{< ref howto-bindings.md >}})
+- [Bindings API reference]({{< ref bindings_api.md >}})
+- [Sample app](https://github.com/harrykimpel/dapr-commercetools-sample) that leverages the commercetools binding
```
Comment (paulyuk) [inline daprdocs/content/en/reference/components-reference/supported-bindings/commercetools.md:71]: Nice sample @harrykimpel, this is useful.

**Review by paulyuk — APPROVED** [review]
Looks good @harrykimpel.  Thank you much for the contribution.  This matches depth and style of the other docs.

**Comment by msfussell** [comment]
Added Code PR https://github.com/dapr/components-contrib/pull/1224.

**Comment by msfussell** [comment]
@harrykimpel - Are you intending on moving forward with these components?

**Pull request closed without merging**